identifiers/country-uk/district_borough_unitary_ward.csv

```diff
@@ -0,0 +1,7306 @@
+ocd-division/country:uk/cty:devon/diw:teign_valley_ward,Teign Valley Ward,E05003614,
+ocd-division/country:uk/cty:devon/diw:bishopsteignton_ward,Bishopsteignton Ward,E05003592,
+ocd-division/country:uk/cty:devon/diw:kenton_with_starcross_ward,Kenton with Starcross Ward,E05003604,
+ocd-division/country:uk/cty:devon/diw:dawlish_central_and_north_east_ward,Dawlish Central and North East Ward,E05003599,
+ocd-division/country:uk/cty:devon/diw:kenn_valley_ward,Kenn Valley Ward,E05003603,
+ocd-division/country:uk/cty:devon/diw:teignbridge_north_ward,Teignbridge North Ward,E05003610,
+ocd-division/country:uk/cty:devon/diw:ipplepen_ward,Ipplepen Ward,E05003602,
+ocd-division/country:uk/cty:devon/diw:college_ward,College Ward,E05008600,
+ocd-division/country:uk/cty:devon/diw:buckland_and_milber_ward,Buckland and Milber Ward,E05003595,
+ocd-division/country:uk/cty:devon/diw:bradley_ward,Bradley Ward,E05003594,
+ocd-division/country:uk/cty:devon/diw:shaldon_and_stokeinteignhead_ward,Shaldon and Stokeinteignhead Ward,E05003609,
+ocd-division/country:uk/cty:devon/diw:teignmouth_east_ward,Teignmouth East Ward,E05003612,
+ocd-division/country:uk/cty:devon/diw:dawlish_south_west_ward,Dawlish South West Ward,E05003600,
+ocd-division/country:uk/cty:devon/diw:bushell_ward,Bushell Ward,E05003596,
+ocd-division/country:uk/cty:devon/diw:teignmouth_west_ward,Teignmouth West Ward,E05003613,
+ocd-division/country:uk/cty:devon/diw:teignmouth_central_ward,Teignmouth Central Ward,E05003611,
+ocd-division/country:uk/cty:devon/diw:bere_ferrers_ward,Bere Ferrers Ward,E05010550,
+ocd-division/country:uk/cty:devon/diw:burrator_ward,Burrator Ward,E05010553,
+ocd-division/country:uk/cty:devon/diw:dartmoor_ward,Dartmoor Ward,E05010555,
+ocd-division/country:uk/cty:devon/diw:bridestowe_ward,Bridestowe Ward,E05010551,
+ocd-division/country:uk/cty:devon/diw:hatherleigh_ward,Hatherleigh Ward,E05010558,
+ocd-division/country:uk/cty:devon/diw:exbourne_ward,Exbourne Ward,E05010557,
+ocd-division/country:uk/cty:devon/diw:drewsteignton_ward,Drewsteignton Ward,E05010556,
+ocd-division/country:uk/cty:devon/diw:buckland_monachorum_ward,Buckland Monachorum Ward,E05010552,
+ocd-division/country:uk/cty:devon/diw:milton_ford_ward,Milton Ford Ward,E05010560,
+ocd-division/country:uk/cty:devon/diw:mary_tavy_ward,Mary Tavy Ward,E05010559,
+ocd-division/country:uk/cty:devon/diw:chagford_ward,Chagford Ward,E05010554,
+ocd-division/country:uk/cty:devon/diw:okehampton_south_ward,Okehampton South Ward,E05010562,
+ocd-division/country:uk/cty:devon/diw:south_tawton_ward,South Tawton Ward,E05010563,
+ocd-division/country:uk/cty:devon/diw:okehampton_north_ward,Okehampton North Ward,E05010561,
+ocd-division/country:uk/cty:devon/diw:tavistock_north_ward,Tavistock North Ward,E05010565,
+ocd-division/country:uk/cty:devon/diw:tavistock_south_west_ward,Tavistock South West Ward,E05010567,
+ocd-division/country:uk/cty:devon/diw:tavistock_south_east_war
```

Comment:
There's an edge case which is not handled here in the parsing code. Note that `Bisley Ward (DET)` and `Bisley Ward` (2 rows up) have the same GSS code (`E05007376`). That is because they're both parts of the same division. (on a map: https://mapit.mysociety.org/area/E05007376.html ). Quote from the BoundaryLine guidance:

> The description ‘Detached, (Det)’, only applies to portions of local government or parliamentary constituency areas which are separated from the main area, being completely surrounded by other local government or parliamentary constituency areas and not connected by direct access on the ground.
> 
> The description will not be applied to islands or parts of islands in the sea.
> 
> Detached parts have the same AI attribute (and GSS code where applicable) as the main area..

so `Bisley Ward` should only have one OCD division identifier. There's a number of other occurrences of this same condition.

Reply:
Looks like we need a special case for that.